hideSocket/hfiles/hardware.h

```diff
@@ -0,0 +1,50 @@
+
```

Comment:
Instead of duplicate this code, share it across the sections

Reply:
Got it👍🏼